SUPPORT.md

```
@@ -0,0 +1,15 @@
+# Support
+
+## Getting Help
+
+- Check [Documentation](./docs)
+- Search [existing issues](../../issues)
+- Join our [Discord/Slack]
+- Email: support@example.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)